controller/src/main/java/io/pravega/controller/store/kvtable/records/KVTConfigurationRecord.java

```diff
@@ -81,8 +81,8 @@ private void read00(RevisionDataInput revisionDataInput,
                                       .kvtName(revisionDataInput.readUTF());
             KeyValueTableConfiguration config = KeyValueTableConfiguration.builder()
                     .partitionCount(revisionDataInput.readInt())
-                    .primaryKeyLength(4) // TODO fix this with https://github.com/pravega/pravega/issues/5939
-                    .secondaryKeyLength(4) // TODO fix this with https://github.com/pravega/pravega/issues/5939
+                    .primaryKeyLength(revisionDataInput.readInt())
```

Comment:
Just FYI. This change is NOT backward compatible. In this particular case, it is OK, as the whole feature is not backward compatible so we do not support "upgrades" from the previous one.

But in another case (or in the future, if we decide to extend this), don't forget to declare a new revision (in `declareRevisions`) and add your data in there.